internal/cli/atlas/clusters/create_test.go

```diff
@@ -45,7 +45,7 @@ func TestCreateOpts_Run(t *testing.T) {
 			members:    3,
 			diskSizeGB: 10,
 			backup:     false,
-			mdbVersion: "4.2",
+			mdbVersion: "4.4",
```

Comment:
I scheduled the cluster tests to make sure there's not an assert expecting this value, good to merge if they pass